src/databricks/labs/ucx/cli.py

```diff
@@ -532,6 +533,22 @@ def join_collection(a: AccountClient, workspace_ids: str):
     account_installer.join_collection(w_ids)
 
 
+@ucx.command
+def upload(
```

Comment:
Note that I on purpose did not add the `a: AccountClient | None` parameter as it will never be used given the current logic to retrieve contexts for multiple workspaces. 
- When settings `a: AccountClient | None` in the signature, it will only be not `None` if `is_account=True` inside the `ucx.command` decorator. However, in that case the `w: WorkspaceClient` points to the accounts endpoints, which does not allow to upload a file when `run_as_collection=False`.
- In the other case, when `AccountClient` is `None`, we are able to use the `w: WorkspaceClient` (`run_as_collection=False`) or get all workspaces in the collection (`run_as_collection=True`) to upload the file to. **Note: in the later, we prompt the user for the account id instead of (possibly) using the account profile from the Databricks configuration as we do when `@ucx.command(is_account=True`). 

There is an issue to merge this logic #2507, with that issue we should improve the integration testing for cross-account compatibility

Reply:
should we keep things simple for the user and create 2 commands i.e. `upload_to_workspace` and `upload_to_account` ?

Reply:
What's the behavior if the file already exists in a workspace ? Shouldn't the user have an `overwrite` flag ? If `overwrite` is `False`, shouldn't the command do a pre-flight check ?

Reply:
There is no option to upload a csv to a Databricks account. We can get the workspaces from the account after which we upload to each workspaces.

For the `overwrite`, that could make sense. Currently we always overwrite. It creates the question on when to break the upload:
- Per workspace, do we **not** upload if the file already exists within that workspace and `overwrite=False`?
- Per collection of workspaces, do we **not** upload the file to **any** workspace if the file already exists within **one** of the workspaces and `overwrite=False`?

Given that we want the command to work on the full collection of workspaces and that all the files ucx uses can be re-generated, I am oke within always overwritting the file to avoid introducing more code complexity